src/com/pty4j/unix/linux/OSFacadeImpl.java

```diff
@@ -86,10 +86,24 @@
   }
 
   // CONSTANTS
+  private static long TIOCGWINSZ;
+  private static long TIOCSWINSZ;
+
+  static {
+    // Initialize architecture-specific constants
+    String arch = System.getProperty("os.arch");
+
+    if (arch.equals("ppc64le") || arch.equals("ppc64")) {
```

Comment:
Should `arch.equals("ppc64")` be checked? There are no binaries for ppc64 arch type.

Reply:
I figured that since the constants were the same for both architectures I should check them both for completeness, but as the patch currently stands the `ppc64` check is useless. I can remove it if necessary.

Reply:
Yes, since the patch targets `ppc64le` please omit checking for `ppc64`. You can add a comment here that constants are the same for `ppc64`.